src/server.c

```diff
@@ -5934,19 +5948,19 @@ void dismissClientMemory(client *c) {
         void *bulk = listNodeValue(ln);
         /* Default bulk size is 16k, actually it has extra data, maybe it
          * occupies 20k according to jemalloc bin size if using jemalloc. */
-        if (bulk) dismissMemory(bulk);
+        if (bulk) dismissMemory(bulk, 0);
     }
 
     /* The 'client' is a bit big because it has 16k default reply buffer. */
-    dismissMemory(c);
+    dismissMemory(c, 0);
 }
 
 /* In child process, we don't need some memory, but it is much possible
  * to change them because clients' heavy requests. We can free them ASAP. */
 void dismissMemoryInChild(void) {
     /* Replication backlog. */
     if (server.repl_backlog != NULL) {
-        dismissMemory(server.repl_backlog);
+        dismissMemory(server.repl_backlog, 0);
```

Comment:
we also know the size of the backlog.. but it is almost certainly huge..

Reply:
yes, it is almost certainly huge, so i didn't, but to make unified, let me add